arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
  # gains:
  #   joint1:
  #     p: 100
  #     d: 1
  #     i: 1
  #     i_clamp: 1
  #   joint2:
  #     p: 100
  #     d: 1
  #     i: 1
  #     i_clamp: 1
  #   joint3:
  #     p: 100
  #     d: 1
  #     i: 1
  #     i_clamp: 1
  #   joint4:
  #     p: 100
  #     d: 1
  #     i: 1
  #     i_clamp: 1
  #   joint5:
  #     p: 100
  #     d: 1
  #     i: 1
  #     i_clamp: 1
  #   joint6:
  #     p: 100
  #     d: 1
  #     i: 1
  #     i_clamp: 1

  constraints:
    goal_time: 0.6
    stopped_velocity_tolerance: 0
    joint_1: {trajectory: 0.1, goal: 0.1}
    joint_2: {trajectory: 0.1, goal: 0.1}
    joint_3: {trajectory: 0.1, goal: 0.1}
    joint_4: {trajectory: 0.1, goal: 0.1}
    stop_trajectory_duration: 0.5
    state_publish_rate: 25
    action_monitor_rate: 10
